lib/commands/module_create.js

```diff
@@ -46,18 +46,25 @@ module.exports.run = function(JAWS, name, action, runtime, pkgMgr, moduleType) {
  * @param modType
  * @constructor
  */
-function CMD(JAWS, name, action, runtime, pkgMgr, modType) {
+function CMD(JAWS, name, action, runtime, pkgMgr, modType, httpMethod) {
 
   if (!runtime) {
     runtime = 'nodejs';
   }
-  
+
   pkgMgr = pkgMgr || false;
 
   if (!supportedRuntimes[runtime]) {
     throw new JawsError('Unsupported runtime "' + runtime + '"', JawsError.errorCodes.UNKNOWN);
   }
 
+  httpMethod = httpMethod.toUpperCase();
+  var supportedHttpMethods = ['DELETE', 'GET', 'HEAD', 'OPTIONS', 'PATCH', 'POST', 'PUT'];
```

Comment:
@sixlettervariables I thought about that, but since this is only used here, I thought it would be good to keep it closer to the usage site. I could go either way and would be happy to change it if the team feels strongly.